LiskHQ/lisk-desktop
# Display graphical statistics of delegates - closes #2623

### What issue have I solved?

-- #2623 

### How have I implemented/fixed it?

- [x] Create Overview component for delegates tab
- [x] Add logic for parse data for each chart, as the data need to be normalize.
- [x] Add 2 more API calls for the charts (Active and Standby delegates and registered delegates)
- [x] Update monitor/delegates component
- [x] Update monitor/delegates unit test component

### How has this been tested?


1. Run the app.
2. Login to the app using Mainnet or Testnet (Monitor data ONLY works for these networks)
3. Go to Monitor Page
4. Check the Delegates Tab
5. The 3  charts should render properly


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-desktop/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-desktop/blob/development/docs/CSS_GUIDE.md)
